examples/example.spec.ts

```
@@ -22,15 +22,7 @@ test('should load the example', async ({ page }) => {
     const text = msg.text();
     console.log(msg.type(), '>>', text);
 
-    if (
-      msg.type() === 'error' &&
-      // Ignore 404 for MathJax fonts
-      msg
-        .location()
-        .url.match(
-          /js\/output\/chtml\/fonts\/tex-woff-v2\/MathJax_(Zero|Math-Italic|Main-Regular)\.woff$/
-        ) === null
-    ) {
+    if (msg.type() === 'error') {
```

Yeah these ignored error were suggesting something was not correct.